### PR TITLE
Fix internal server error, when removing media with relation from Sylius grid

### PR DESCRIPTION
**Pull request by em411**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | ---
| License         | MIT

### CODE REVIEW

File: src/EventListener/ResourceDeleteSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * another great project.
+ * You can find more information about us on https://bitbag.shop and write us
+ * an email on mikolaj.krol@bitbag.pl.
+ */
```
Comment (damonsson) [inline src/EventListener/ResourceDeleteSubscriber.php:9]: For sure it's old description, there is new one.

File: src/Resources/config/grids/admin/media.yml [hunk 2]
```diff
@@ -7,8 +7,7 @@ sylius_grid:
                     class: "%bitbag_sylius_cms_plugin.model.media.class%"
                     repository:
                         method: createListQueryBuilder
-                        arguments:
-                            - "expr:service('sylius.context.locale').getLocaleCode()"
+                        arguments: ["%locale%"]
```
Comment (damonsson) [inline src/Resources/config/grids/admin/media.yml:10]: Where is %locale% from? Will we still have a dynamic language depending on the current context?

File: src/EventListener/ResourceDeleteSubscriber.php [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+        return new RedirectResponse($this->router->generate($redirectRoute));
+    }
+
+    private function isMethodDelete(Request $request): bool
+    {
+        return Request::METHOD_DELETE === $request->getMethod();
+    }
+
+    private function isBitBagRoute(string $route): bool
```
Comment (damonsson) [inline src/EventListener/ResourceDeleteSubscriber.php:114]: maybe isProtectedRoute?

File: src/Resources/config/grids/admin/media.yml (hunk 2 above)
Comment (em411) [reply src/Resources/config/grids/admin/media.yml:10]: I've changed this to stay consistent with other grids in the CMS plugin and grids from the Sylius, it's coming from Sylius internals, and we will still have a dynamic language depending on the current context.

File: src/EventListener/ResourceDeleteSubscriber.php (hunk 1 above)
Comment (em411) [reply src/EventListener/ResourceDeleteSubscriber.php:9]: I've updated the new description in BitBagBible, but I've decided to add the old one here to stay consistent with the rest of the plugin

**Pull request merged**